src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -234,7 +235,137 @@ TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_SUCCESS) {
             (*command_msg)[am::strings::msg_params][am::hmi_response::urls][0]
                           [strings::url].asString());
 }
+TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_IncorrectIdForSending_UNSUCCESS) {
+  MessageSharedPtr command_msg(
+      CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
+  (*command_msg)[am::strings::msg_params][am::strings::number] = "123";
```

Comment:
@dtrunov Well I have added it since I was comparing my units with similar units from this class, however now I see that this field is compleately unused in all of the units, should I deleate it then? if yes, should I delete them from my or from all test units in this class?